platform/android/java/app/AndroidManifest.xml

```diff
@@ -20,6 +20,8 @@
         android:glEsVersion="0x00020000"
         android:required="true" />
 
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.INTERNET" />
 <!-- Custom user permissions XML added by add-ons. It's recommended to add them from the export preset, though. -->
```

Comment:
Shouldn't this be optional? When you are building a normal release version you don't want this in the manifest?